abstract.js

```diff
@@ -624,6 +624,7 @@ function abstractPersistence (opts) {
       var stream = instance.outgoingStream(client)
 
       stream.pipe(concat(function (list) {
+        list[0].messageId = updated.messageId
```

Comment:
list[0] also has `messageId = 0`, so then we should also delete it from list[0] @mcollina